app/components/search_result_component/search_result_component.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module SearchResultComponent
+  class SearchResultComponent < ApplicationComponent
```

Comment:
I know the component generator is currently appending `component` to everything – to be consistent with how the other components are named, could you rename the files/class? (I’ll push an updated generator!)

`search_result_component/search_result_component.rb` --> `search_result/search_result.rb`

```suggestion
module SearchResult
  class SearchResult< ApplicationComponent
```